tests/ui_json_utils_test.py

```diff
@@ -264,3 +265,21 @@ def test_is_form_test():
     assert is_form(ui_json["string_parameter"])
     ui_json["string_parameter"].pop("value")
     assert not is_form(ui_json["string_parameter"])
+
+
+def test_flatten_group_value():
+    ui_json = deepcopy(default_ui_json)
+    ui_json["test"] = {
```

Comment:
Can you add a template for it under `ui_json.template` like the other ones, and use it here?